.github/workflows/trigger-metamask-institutional-e2e-ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Trigger MetaMask Institutional E2E CI
+
+on:
+  push:
+    branches: [ Version-v*, master ]
+  pull_request:
+    branches: [ Version-v*, master ]
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  trigger-mmi-e2e-ci:
+    runs-on: ubuntu-latest
+    if: ${{ (!github.event.pull_request.draft && (startsWith(github.head_ref, 'Version-v') || github.head_ref == 'master')) || (!github.event.pull_request) }}
+    steps:
+      - name: Trigger MetaMask Institutional E2E CI
+        uses: peter-evans/repository-dispatch@v2
```

Comment:
The MetaMask organization has a global allowlist for third-party actions. We could pursue getting this added to that list, but as an alternative have you considered using `curl` directly? As demonstrated in the docs: https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event

This would require no organization action allowlist change

Reply:
Ok, that make sense